SP-2019: Add description of Solar System data products in DP1 (#45)
lsst/rtn-011: datapreview.tex

A series of three Data Previews (DP) and one Data Release (DR) are planned to support the community as they develop their LSST analysis software and workflows, and to enable high-impact science as soon as possible.
\begin{itemize}
\item Data Preview 0 (DP0): Based on simulated LSST-like data.
\item Data Preview 1 (DP1): Based on a subset of early science-grade commissioning data taken with LSSTComCam during commissioning.
\item Data Preview 2 (DP2): Based on a full reprocessing of all science-grade data taken with LSSTCam during commissioning.
\item Data Release 1 (DR1): Based on the first six months of LSST data. 
\end{itemize}

> Due to the relatively short time periods available for commissioning observations (\S~\ref{ssec:transition}), the Data Previews will necessarily be limited in their area and temporal coverage relative to a full Data Release. 

Reviewer: Suggesting moving all these to the past tense.

> DP0.1 only uses the 300~\sqdeg of DC2 images that were simulated for five years of the LSST’s wide-fast-deep component (WFD) using a baseline cadence,~\citedsp{PSTN-055}.
> The DESC processed the simulated DC2 images with~\href{https://pipelines.lsst.io/v/v19_0_0/index.html}{Version 19} of the LSST Science Pipelines.
> DP0.1 makes the DESC’s DC2 images and catalogs available to users through an early version the Rubin Science Platform (RSP) running at the US DAC. 

Reviewer: ```suggestion
DP0.1 used the 300~\sqdeg of DC2 images that were simulated for five years of the LSST’s wide-fast-deep component (WFD) using a baseline cadence,~\citedsp{PSTN-055}.
The DESC processed the simulated DC2 images with~\href{https://pipelines.lsst.io/v/v19_0_0/index.html}{Version 19} of the LSST Science Pipelines, producing calibrated images and catalogs.
DP0.1 made the DESC’s DC2 images and catalogs available to users through an early version of the Rubin Science Platform (RSP) running at the US DAC.
DP0.1 has now been retired from the publicly available RSP instance.
```

> For full details on DP0.1 including an exact description of the data products served, see the documentation at \url{https://dp0-1.lsst.io/}

Reviewer: ```suggestion
For full details on DP0.1 including an exact description of the data products that were served, see the documentation at \url{https://dp0-1.lsst.io/}.
```

> The dataset used for DP0.2 was the same as that used for DP0.1.

Reviewer: ```suggestion
The input simulated image dataset, DESC's ``DC2,'' used for DP0.2 was the same as that used for DP0.1.
```

> Rubin processed the simulated DC2 images with~\href{https://pipelines.lsst.io/v/v23_0_0/index.html}{Version 23} of the LSST Science Pipelines.

Reviewer: ```suggestion
In this case, the Rubin team itself processed the simulated DC2 images with~\href{https://pipelines.lsst.io/v/v23_0_0/index.html}{Version 23} of the LSST Science Pipelines.
```

DP0.2 makes the Rubin reprocessed DESC DC2 images and catalogs available to users through an early version the Rubin Science Platform (RSP) running at the US DAC. 

> For full details on DP0.2 including an exact description of the data products served, see the documentation at \url{https://dp0-2.lsst.io/}

Reviewer: ```suggestion
For full details on DP0.2 including an exact description of the data products served, see the documentation at \url{https://dp0-2.lsst.io/} .
```

Only 1x30 second exposure images are included in DP1.
\input{tables/dp1_fields_visits}
\input {figures/dp1_fields_depths}

The  processing and preparation of LSSTComCam data for DP1 took place during the first half of 2025. 
The planned release date for DP1 is 30 June 2025. 
The planned data products for DP1 are presented in Table~\ref{tab:data-preview-summary}.
DP1 is expected to include of the order of 2000 exposures.
The processing of crowded fields in the centers of 47 Tuc and Fornax is on a best-effort basis.
The center of 47 Tuc is saturated and so no coadd images are expected to be produced for this region.
Solar System catalogs will contain observations of known objects from the MPC Orbit Catalog (MPCORB) tables as well as any new discoveries. 
The snapshot of the MPCORB Catalog tables included in DP1 is taken from around the time of DRP processing for DP1. 

As the processing progresses, it is conceivable that some individual exposures might not meet the quality control standard of being ``science-grade, astronomically useful'' images, 

> and consequently, the exact number of exposures and coadd depths in DP1 may change.

Reviewer: ```suggestion
and consequently, the exact number of exposures and coadd depths in DP1 may change before the actual release.
```

(We're not trying to claim it will change after the release, I hope?)

Additionally, some columns may be missing from some catalogs due to the small data sets involved in DP1.
There will be no cell-based coadds in DP1, nor \textit{ShearObject} catalog.
Both of these are currently stretch goals for DP2.
		
\subsection{Data Preview 2}
\label{ssec:dp2}

> As the survey begins, all science-grade data collected during the commissioning System Optimization period and subsequent Science Validation Surveys, \S~\ref{sec:commissioning}, is reprocessed to produce the final Data Preview, DP2, which is expected to be released approximately 6 months following the start of operations.

Reviewer: ```suggestion
As the survey begins, all science-grade LSSTCam data collected during the commissioning System Optimization period and subsequent Science Validation Surveys, \S~\ref{sec:commissioning}, will be reprocessed to produce the final Data Preview, DP2, which is expected to be released approximately 6 months following the start of operations.
```

No coadds were produced for these regions during the on-sky commissioning campaign, hence the holes in the center of the figures.
A typical observing epoch on a given target field consisted of 5-20 visits in each of the three loaded filters.
Nearly all the visits  were taken with one single 1x30 second exposure time and not  2x15 second snaps\footnote{At this time, studies as to whether the LSST will use a 1x30 single exposure or 2x15 snaps as the default standard visit are ongoing with LSSTCam data, and a decision will be made prior to starting the LSST. }.
\input{tables/dp1_fields_visits}
\input {figures/dp1_fields_depths}

The  processing and preparation of ComCam data for DP1 will take place during the first half of 2025, with an expected DP1 release data in June or July 2025. 
The planned data products for DP1 are presented in Table~\ref{tab:data-preview-summary}.
DP1 is expected to include of the order of 2000 science images.
The processing of crowded fields in the centers of 47 Tuc and Fornax is on a best-effort basis.
The center of 47 Tuc is saturated and so no coadd images will be produced for this region.

> Solar System catalogs will contain observations of known objects from the MPC Orbit (MPCORB) tables.

Reviewer 2: ```suggestion
Solar System catalogs will contain observations of known objects from the MPC Orbit Catalog (MPCORB) tables.
```
